.github/reviewer-lottery.yml

```diff
@@ -0,0 +1,20 @@
+groups:
+  # Default reviewers for all pull requests.
+  # Usually, at least on of the maintainers should approve PR before merging.
+  # The best is if two maintainers do that.
+  - name: maintainers # name of the group
+    reviewers: 2 # how many reviewers do you want to assign?
+    internal_reviewers: 1 # how many reviewers do you want to assign when the PR author belongs to this group?
+    usernames: # github usernames of the reviewers
+      - keiya01
+      - kaWaite
```

Comment:
```suggestion
      - KaWaite
```

Reply:
??

Reply:
I guess there is a chance it is case-sensitive. My github username is KaWaite, not kaWaite 👍🏻

Reply:
Ohh Damn, Thanks for catching that @keiya01 !